### ISSUE TITLE
Changing email address of an existing member?

### ISSUE THREAD

**Issue by andredasilvapinto** [Ecwid/maleorang#10]
Shouldn't this be marked as a field too:

https://github.com/Ecwid/maleorang/blob/master/src/main/java/com/ecwid/maleorang/method/v3_0/lists/members/EditMemberMethod.kt#L36

in order to conform to:

http://developer.mailchimp.com/documentation/mailchimp/reference/lists/members/#edit-patch_lists_list_id_members_subscriber_hash

?

**Comment by basiliscus** [comment]
Hi @andredasilvapinto ,

Maybe... but what changes if we do so? It seems, nothing.

Probably, we should move the `email_address` field from the constructor to the class body, and make it mutable. That will probably let us modify the subscriber's email. Not sure of that though, an experiment is needed.

**Comment by andredasilvapinto** [comment]
yes, that's what I meant. email_address should be a field. it does change the email in that case.

**Comment by yuvalfar** [comment]
@basiliscus I can confirm making it mutable should work. the idea is that subscriber_hash should stay the hash of the old email (coming from the constructor). then the caller can change the field email_address on the method and update subscriber's email.

The use case is pretty common, users changing their emails from time to time and ideally change on a platform will reflect in Mailchimp.

Also see https://stackoverflow.com/questions/32224697/mailchimp-api-v3-0-change-subscriber-email discussion, the change from November 2016 added this ability to the API

How can we get that pull request approved?
